src/ethereum/index.js

```diff
@@ -44,7 +58,8 @@ export function loadContracts() {
     },
     LANDRegistry: {
       address: getEnvAddress('LAND_REGISTRY_CONTRACT_ADDRESS'),
-      eventNames: ['Update', 'Transfer', 'UpdateOperator']
+      eventNames: ['Update', 'Transfer', 'UpdateOperator'],
+      types: { Transfer: 2 }
```

Comment:
I think this is a good solution but the name `types` and `getNumberTypesOfEvents` is quite confusing. We talked about `amount` or `count`, you think those are better options?